resource-managers/kubernetes/integration-tests/src/test/scala/org/apache/spark/deploy/k8s/integrationtest/DepsTestsSuite.scala

```diff
@@ -45,6 +48,7 @@ private[spark] trait DepsTestsSuite { k8sSuite: KubernetesSuite =>
   val BUCKET = "spark"
   val ACCESS_KEY = "minio"
   val SECRET_KEY = "miniostorage"
+  val REGION = "us-west-2"
```

Comment:
How does v1 work without REGION?

Reply:
When initializing a v2 S3 client, it is required to specify a region (either via codes, environment variables or system properties etc). 
But when initializing a v1 S3 client, region is not mandatory. For instance, I was able to list all buckets across regions with the following.
```
    val s3client = new AmazonS3Client()
    val response1 = s3client.listBuckets()
    println(response1)
```

Reply:
the whole region thing is a real pain...for AWS itself you are now expected to declare the region and have it work everything out; for third party just the endpoint and any region string should suffice. See [HADOOP-18908](https://issues.apache.org/jira/browse/HADOOP-18908) for our ongoing struggles there